ruby-on-rails/app/controllers/benchmark_controller.rb

```diff
@@ -0,0 +1,5 @@
+class BenchmarkController < ApplicationController
+  def ping
+    puts "123"
```

Comment:
This should be `render plain: '123'`. I'm assuming all thats needed is just a plain text response